### PR TITLE
Added test for servlet wrapper

### LINKED ISSUE #289: Servlet wrappers for
Let's create servlet wrappers to use `Slice` implementations in servlet-based web applications such as spring-mvc.

### PR DESCRIPTION
**Pull request by g4s8**
Closes #289 - added integration test for servlet slice wrapper
based on embedded Jetty server.

### CODE REVIEW

File: src/test/java/com/artipie/http/servlet/ServletWrapITCase.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+    @Test
+    void simpleSliceTest() throws Exception {
+        final String text = "Hello servlet";
+        this.start(new SliceSimple(new RsText(text)));
+        final String body = HttpClient.newHttpClient().send(
+            this.req.copy().GET().build(),
+            HttpResponse.BodyHandlers.ofString()
+        ).body();
+        MatcherAssert.assertThat(body, Matchers.equalTo(text));
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/http/servlet/ServletWrapITCase.java:95]: @g4s8 in our contributing guide we [demand](https://github.com/artipie/artipie/blob/master/CONTRIBUTING.md#code-style) not use `Matchers` static methods, I'd suggest to correct it in the test

**Review by olenagerasimova — APPROVED** [review]
@g4s8 one suggestion, take a look please

**Comment by g4s8** [comment]
@olenagerasimova thanks, should be fixed

File: src/test/java/com/artipie/http/servlet/ServletWrapITCase.java [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    @Test
+    void echoSliceTest() throws Exception {
+        this.start((line, headers, body) -> new RsWithBody(body));
+        final String test = "Ping";
+        final String body = HttpClient.newHttpClient().send(
+            this.req.copy().PUT(HttpRequest.BodyPublishers.ofString(test)).build(),
+            HttpResponse.BodyHandlers.ofString()
+        ).body();
+        MatcherAssert.assertThat(body, new IsEqual<>(body));
+    }
```
Comment (genryxy) [inline src/test/java/com/artipie/http/servlet/ServletWrapITCase.java:108]: @g4s8 I think a typo here because we need to compare with `test`

**Comment by g4s8** [comment]
@genryxy thanks, fixed

**Review by genryxy — APPROVED** [review]
@g4s8 thanks, looks good

**Comment by 0crat** [comment]
Job `gh:artipie/http#291` is not assigned, can't get performer

**Pull request merged**